### PR TITLE
Customize inference widget input

### PR DESCRIPTION
**Pull request by mrm8488**

### CODE REVIEW

File: model_cards/mrm8488/RuPERTa-base/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+language: spanish
+widget:
+- text: "España es un país my <mask> en UE"
```
Comment (JetRunner) [inline model_cards/mrm8488/RuPERTa-base/README.md:4]:
```suggestion
```

**Review by JetRunner — APPROVED** [review]
Actually, we have a default Spanish example:
```
Mi nombre es <mask> y soy un ingeniero que vive en Nueva York.
```
so you only need to specify the language here.

File: model_cards/mrm8488/RuPERTa-base/README.md (hunk 1 above)
Comment (mrm8488) [reply model_cards/mrm8488/RuPERTa-base/README.md:4]: Feel free to change it if you want. In fact I made a typo: I meant "España es un país muy <mask> en UE", not "my"

**Comment by julien-c** [comment]
@JetRunner There's still to possibility to override the default inputs if it makes more sense for the model: https://twitter.com/mrm8488/status/1282778743598194688

**Comment by julien-c** [comment]
(up to @mrm8488 if it does here)

**Comment by mrm8488** [comment]
In this case it makes more sense because in last phase the model was fine tuned on Spanish Wikipedia data

**Pull request merged**